docs/tags.rst

```diff
@@ -0,0 +1,114 @@
+Tags
+====
+
+.. currentmodule:: packaging.tags
+
+Wheels encode the Python interpreter, ABI, and platform that they support in
+their filenames using *tags*. `PEP 425`_ specifies how to interpret these tags
+as well as how to specify them in a wheel filename.
```

Comment:
https://packaging.python.org/specifications/platform-compatibility-tags/ is the stable reference for the tagging scheme (it links to the manylinux PEPs in addition to PEP 425)